.github/workflows/main.yaml

```diff
@@ -87,6 +87,8 @@ jobs:
             container-image: "golang"
             enable_arm64: true
             request-scan: "true"
+          - dir: "./grafana"
+            container-image: "grafana"
```

Comment:
Probably, grafana should be scanned because the base image is ubuntu (not scratch).

Reply:
Currently scan requirement is auto-detected, so I think there would be no problem.
ref: https://github.com/cybozu/neco-containers/actions/runs/7828640467/job/21358962831#step:4:178

Reply:
I see, it looks good as is :)